azext_iot/central/params.py

```diff
@@ -30,23 +30,67 @@
     options_list=["--style"],
     choices=CaseInsensitiveList(["scroll", "json", "csv"]),
     help="Indicate output style"
-    "scroll = deliver errors as they arrive, json = summarize results as json, csv = summarize results as json",
+    "scroll = deliver errors as they arrive, json = summarize results as json, csv = summarize results as csv",
 )
 
 
 def load_central_arguments(self, _):
     """
     Load CLI Args for Knack parser
     """
-    with self.argument_context("iot central app") as context:
+    with self.argument_context("iot central") as context:
         context.argument("app_id", options_list=["--app-id", "-n"], help="Target App.")
-        context.argument("minimum_severity", arg_type=severity_type)
+        context.argument(
+            "token",
+            options_list=["--token"],
+            help="Authorization token for request. "
+            "More info available here: https://docs.microsoft.com/en-us/learn/modules/manage-iot-central-apps-with-rest-api/ "
+            "MUST INCLUDE type (e.g. 'SharedAccessToken ...', 'Bearer ...'). "
+            "Example: 'Bearer someBearerTokenHere'",
+        )
+        context.argument(
+            "central_dns_suffix",
+            options_list=["--central-dns-suffix", "--central-api-uri"],
+            help="Central dns suffix. "
+            "This enables running cli commands against non public/prod environments",
+        )
+
+    with self.argument_context("iot central device-template") as context:
+        context.argument(
+            "device_template_id",
+            options_list=["--device-template-id", "--dtid"],
+            help="Device template id. Example: somedevicetemplate",
+        )
+        context.argument(
+            "content",
+            options_list=["--content", "-k"],
```

Comment:
Awesome thanks!!

One minor last fix: try to dedup the last few params

I'm seeing two instances of --content (in device and device_template) and --role (in user and in api-token)

You can move these into the central base params

Reply:
wont fix - pattern similar to other areas